src/main/java/org/sunbird/workflow/config/Constants.java

```diff
@@ -316,5 +316,15 @@ private Constants() {
 	public static final String SENDER = "sender";
 	public static final String TRANSFER_SUBJECT_LINE = "Transfer Notification: #username";
 	public static final String LAST_UPDATED_ON = "lastUpdatedOn";
+	public static final String DOMICILE_MEDIUM = "domicileMedium";
+	public static final String PINCODE = "pinCode";
+	public static final String EMPLOYEE_CODE = "employeeCode";
+	public static final String PROFILE_STATUS = "profileStatus";
+	public static final String NOT_VERIFIED = "NOT-VERIFIED";
+	public static final String GENDER = "gender";
+	public static final String DOB = "dob";
+	public static final String CATEGORY = "category";
+	public static final String EXTERNAL_SYSTEM_ID = "externalSystemId";
+	public static final String EXTERNAL_SYSTEM = "externalSystem";
```

Comment:
If we are not using any the new keys -- then we don't need to add  - NOT_VERIFIED is not used anywhere ?